code/game/objects/effects/contraband.dm

```diff
@@ -1015,6 +1020,47 @@
 	/obj/structure/sign/poster/contraband/d_day_promo,
 		)
 
+//RILENA poster pool. There are only five of these, so try not to go overboard with the random posters, okay? Expect not to see these mapped onto ships except in rolled up form, since they're supposed to be bought in the RILENA merch crate.
```

Comment:
What if we gave this an art element that only affected Rilena fans, sort of like the Karl Marx bust did for revs? Potentially out of scope.

Example:
```dm
/datum/component/art/rilena

/datum/component/art/rilena/apply_moodlet(mob/fan, impress)
	fan.visible_message(
		"<span class='notice'>[fan] stops to inspect [parent].</span>",
		"<span class='notice'>You take in [parent].</span>"
	)

	if(HAS_TRAIT(L, TRAIT_FAN_RILENA))
		SEND_SIGNAL(M, COMSIG_ADD_MOOD_EVENT, "artok", /datum/mood_event/artok)
```

and then just apply it to the posters or other merch, possibly

Reply:
I have no clue how to really do this but I can try

Reply:
I did it!